content/en/blog/_posts/2021-04-08-introducing-indexed-jobs.md

```diff
@@ -0,0 +1,90 @@
+(for now):
+
+- `NonIndexed` (default): the Job is considered complete when there have been
+  a number of successfully completed Pods equal to the specified number in
+  `.spec.completions`. In other words, each Pod completion is homologous to
+  each other. Any Job you might have created before the introduction of
+  completion modes is implicitly NonIndexed.
+- `Indexed`: the Job is considered complete when there is one successfully
+  completed Pod associated to each index from 0 to `.spec.completions-1`. Each
+  Pod gets its associated index in the annotation
+  `batch.kubernetes.io/job-completion-index`.
+
+You can start using Jobs with Indexed completion mode, or Indexed Jobs, for
+short, to easily start parallel Jobs in which each worker works on a partition
+of the data that is statically assigned. This saves you from having to set up
+a queuing system or even having to modify your binary!
+
+## Creating an Indexed Job
+
+To create an Indexed Job, you just have to add `completionMode: Indexed` to the
+Job spec and make use of the `JOB_COMPLETION_INDEX` environment variable.
+
+```yaml
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: 'sample-job'
+spec:
+  completions: 3
+  parallelism: 3
+  completionMode: Indexed
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+      - command:
+        - 'bash'
+        - '-c'
+        - 'echo "My partition: ${JOB_COMPLETION_INDEX}"'
+        image: 'docker.io/library/bash'
+        name: 'sample-load'
+```
+
+When you run the example, you will see that each of the 3 created Pods gets a
+different completion index. For users convenience, the control plane sets the
+`JOB_COMPLETION_INDEX` environment variable, but you can choose to [set your
+own](/docs/tasks/inject-data-application/environment-variable-expose-pod-information/)
+or [expose the index as a file](/downward-api-volume-expose-pod-information/).
+
+See [Indexed Job for parallel processing with static work
+assignment](/docs/tasks/job/indexed-parallel-processing-static/) for a
+step-by-step guide, and a few more examples.
+
+## Future plans
+
+We envision that there might be more completion modes that enable more use cases
+for the Job API. We welcome you to open issues in
+[kubernetes/kubernetes](https://github.com/kubernetes/kubernetes) with your
+suggestions.
+
+In particular, we are considered an `IndexedAndUnique` mode were the indexes
+are not just available as annotation, but they are part of the Pod names,
+similar to {{< glossary_tooltip text="StatefulSet" term_id="statefulset" >}}.
+This should facilitate inter-Pod communication for tightly coupled Pods.
+You can join the discussion in the [open issue](https://github.com/kubernetes/kubernetes/issues/99497).
+
+## Wrap-up
+
+Indexed Jobs allows you to statically partition work among the workers of your
+parallel Jobs. We hope that this feature facilitates the migration of more batch
```

Comment:
```suggestion
parallel Jobs. I hope that this feature facilitates the migration of more of your batch
```
?

only because there's just 1 author, and with https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we in mind (from the documentation style guide, mind).

Reply:
Can I say "SIG Apps hopes..."?

Any recommendation for the 2nd paragraph that avoid either "we" or "I"?

Reply:
Yep, “SIG Apps hopes” works well.